src/governance-object.cpp

```diff
@@ -515,7 +521,7 @@ CAmount CGovernanceObject::GetMinCollateralFee()
     }
 }
 
-bool CGovernanceObject::IsCollateralValid(std::string& strError)
+int CGovernanceObject::GetCollateralConfirmations(std::string& strError)
```

Comment:
I have committed suggested changes `ec33faa273cf4b3e8ac74acdb7d5cec14ed98a54`. If it is better I can merge these changes to previous commit.